cruise-control-core/src/main/java/com/linkedin/cruisecontrol/servlet/parameters/CruiseControlParameters.java

```diff
@@ -19,13 +19,12 @@ public interface CruiseControlParameters extends CruiseControlConfigurable {
 
   /**
    * Parse the parameters set in HTTP request to initialize object.
+   * @param handler The request handler
    *
-   * @param response HTTP response of Cruise Control. Populated in case of a parameter parse exception.
    * @return {@code true} if there has been a failure to parse parameters, {@code false} otherwise. If the object is already initialized,
    *         directly return false.
    */
-  boolean parseParameters(HttpServletResponse response);
-
+  boolean parseParameters(CruiseControlRequestContext handler);
```

Comment:
The param naming here is a big confusing. From naming conversion, either the class should call something like "CruiseControlRequestHandler", or the param should call "requestContext"